dynamic-simulation/dynamic-simulation-api/src/main/java/com/powsybl/dynamicsimulation/DynamicSimulation.java

```diff
@@ -113,9 +140,9 @@ public static Runner find() {
         return find().runAsync(network);
     }
 
```

Comment:
We also need all the run and runAsync combinations in this part of the file please.